cmd/options.go

```diff
@@ -89,6 +90,12 @@ func optionFlagSet() *pflag.FlagSet {
 	flags.StringSlice("tag", nil, "add a `tag` to be applied to all samples, as `[name]=[value]`")
 	flags.String("console-output", "", "redirects the console logging to the provided output file")
 	flags.Bool("discard-response-bodies", false, "Read but don't process or save HTTP response bodies")
+	flags.String("ui-mode", UIModeResponsive.String(),
```

Comment:
Hhmm true. I suppose we can detect if we're running in non-interactive mode easily enough and adapt accordingly, but can we rely on all CIs to run k6 non-interactively?

Having an option makes this easier to control / override regardless of the environment, which seems safer. That said, I also can't think of a solid reason for turning off responsiveness if running in an interactive terminal, unless someone is annoyed by it or it doesn't work great in their terminal.

Reply:
I think it is better for there to be the option to disable/enable it

Reply:
Again, this has nothing to do with whether we're running on a CI or not, this is determined here https://github.com/loadimpact/k6/blob/92a9716e6b3ac3574cd9a449e0afdad584c75b5a/cmd/ui.go#L250-L255

That said, I'm not strictly against having such an option, I just don't see a lot of reasons why someone would choose `full` or `compact` over `responsive`. 

I think there would be a lot more value in this option, if we had `log` as one of the possible values. Then, instead of any progressbars, k6 can just emit execution information as log messages, making it even more suitable for CI. Ideally (post-#883), it'd be nice to have a way to specify that 1s/100ms refresh rate I linked to above, say `--ui-mode="log,5s"` or something like that.

Reply:
Hey, what can we decide here?

I'm not against having responsive mode always, and repurposing `--ui-mode` for changing how output is handled. I like being able to specify log-mode only, but maybe the refresh rate could be a separate option, which would work for all "UI modes", and we can avoid the string splitting. Something like `--ui-mode=log --ui-refresh-rate=5000`. But that's definitely for another PR, so create an issue for it?

In case we go for the above, I would remove `--ui-mode` from this PR, but that still leaves users with no way to force compact mode, for CI scenarios, which was the main reason for adding it.

@na-- Can we rely on that to always be false in all CI environments? It seems like having explicit control over this behavior would be safer.